yarn.lock

```diff
@@ -2,24 +2,17 @@
 # yarn lockfile v1
```

Comment:
Is there any special reason to change this file?

Reply:
No, I do not know why this file changed

Reply:
please revert all unnecessary changes. All changes in a PR should be intentional.

Reply:
I know that, but I want to understand why it happened

Reply:
I have no idea why. Please just revert it.

Reply:
I tried to do it when I pushed the revert for the other file, but yarn. lock just regressed to its current state.